games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
+  "description": "The well known game. Destroy the invaders with your ship.",
+  "images": [
+    "./Space Invaders [David Winter] - main screen.png",
+    "./Space Invaders [David Winter] - game play.png"
+  ],
+  "interpreter": {
+    "id": "chip8-david-winter",
+    "name": "CHIP8 by David Winter"
+  },
+  "binaries": {
+    "names": [
+      { "value": "Space Invaders [David Winter] - 0.9.1" },
+      { "value": "Space Invaders [David Winter]" },
+
+      { "value": "Space Invaders [David Winter] - 0.9.0" },
+      { "value": "Space Invaders [David Winter] (alt)" },
+      { "value": "INVADERS" }
+    ],
+    "extensions": [
+      "c8",
+      "ch8",
+      ""
+    ],
+    "hashes": [
+      {
+        "type": "SHA256",
+        "value": "00bb7001de52b562d98357d33375406c6d0308466d7d95c2580537d2451a75cd"
+      },
+      {
+        "type": "SHA256",
+        "value": "2d0e1fa53216b297e74041d4fb766f42327a42893e83bb4ec931a9dff5c2dd10"
+      }
```

Comment:
Hash is an object with type (I used SHA256 since it's more secure than SHA1) and the value. I guess it would be best to agree on one type so emulators don't have to support many.

Hashes are included for companion file case.

Reply:
Security isn't really a factor here, is it? And collisions will be very rare regardless. That was at least the reason I originally picked one that is very widely supported and computationally inexpensive. Doesn't really matter to me though, as long as it's standardized.

Reply:
Let's use SHA1 then.

Reply:
Moved hashes into binaries section. They are now SHA1.